lib/inc/internal/ruby/module.hpp

```diff
@@ -92,6 +92,14 @@ namespace facter { namespace ruby {
          */
         facter::facts::collection& facts();
 
+        /**
+         * Gets hide_legacy.
+         * @return Returns boolean.
+         */
+         bool hide_legacy(){
```

Comment:
header should contain only declaration `bool hide_legacy() const;`, while corresponding cpp file should contain the definition.
also const should be used if class is not changed